extensions/quarkiverse-mcp-server-sse.yaml

```diff
@@ -0,0 +1,6 @@
+---
+group-id: "io.quarkiverse.langchain4j"
```

Comment:
Wrong `goupId`? It should be `io.quarkiverse.mcp`...

Reply:
Oop[s